CMakeLists.txt

```diff
@@ -114,38 +124,38 @@ target_include_directories(mdspan INTERFACE
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include>
 )
+target_compile_features(mdspan INTERFACE cxx_std_${CMAKE_CXX_STANDARD})
 
 ################################################################################
 
-install(TARGETS mdspan EXPORT mdspanTargets
+install(TARGETS mdspan EXPORT MDSpanTargets
```

Comment:
No I think `mdspan` was right if the package name is "mdspan".
The intent is user will do `find_package(mdspan)` and not `find_package(MDSpan)`

Reply:
The project name is MDSpan, the library name is mdspan and the namespace is mdspan.
Normale the project name and the namespace is the same. But IMHO, it must not,
see https://cmake.org/cmake/help/latest/command/find_package.html?highlight=find_package#id3